_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -361,6 +361,117 @@ public function show_setup_wizard() {
 		 */
 		return apply_filters( 'jetpack_show_setup_wizard', false ) && Jetpack::is_active();
 	}
+
+	/**
+	 * Returns the release post content and image data as an associative array.
+	 * This data is used to create the update modal.
+	 */
+	public function get_update_modal_data() {
+		$post_data = $this->get_release_post_data();
+
+		if ( ! isset( $post_data['posts'][0] ) ) {
+			return;
+		}
+
+		$post = $post_data['posts'][0];
+
+		$post_content = isset( $post['content'] ) ? $post['content'] : null;
+		if ( empty( $post_content ) ) {
+			return;
+		}
```

Comment:
Maybe we could simplify this a bit, since we're only using `$post_content` once later and since the `empty` check takes care of checking if `isset`?

```suggestion
		if ( empty( $post['content'] ) ) {
			return;
		}
```

Reply:
Good catch! I'll change that.